force-app/main/default/classes/DonationHistoryController_TEST.cls

```diff
@@ -0,0 +1,87 @@
+@IsTest
```

Comment:
All classes require the same copyright and info headers

Reply:
missing copyright added!